### PR TITLE
🌟 add support of workspace networkpolicy

### PR DESCRIPTION
**Pull request by magicsong**

### CODE REVIEW

File: config/samples/network_v1alpha1_workspacenetworkpolicy.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: network.kubesphere.io/v1alpha1
+kind: WorkspaceNetworkPolicy
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: workspacenetworkpolicy-sample
+spec:
+  # Add fields here
+  foo: bar
```
Comment (zheng1) [inline config/samples/network_v1alpha1_workspacenetworkpolicy.yaml]: Is it wrong to input `foo` as key?

Comment (magicsong) [reply config/samples/network_v1alpha1_workspacenetworkpolicy.yaml]: it is a generated file. would move it out of repo

File: .travis.yml [hunk 2]
```diff
@@ -10,6 +10,10 @@ go:
   - "1.12.x"
 env:
   - GO111MODULE=on
+cache:
+  directories:
+    - $HOME/Library/Caches/go-build
```
[the remark this answers is not in the document]
Comment (magicsong) [reply .travis.yml]: done, ptal

**Pull request merged**